.github/workflows/ready-to-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: 🤖 PR is ready to merge
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  label-when-approved:
+    name: Label when approved
+    runs-on: ubuntu-latest
+    if: github.event.review.state == 'APPROVED'
```

Comment:
Should it be lowercased ?

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#running-a-workflow-when-a-pull-request-is-approved

Reply:
fixed